about.html

```diff
@@ -0,0 +1,127 @@
+        The world's largest livestock sale, with over 40,000 cattle sold annually, as well as 200 million litres of milk. And including a sparkling lineup of the biggest names in the production of beef animals and dairy herds.
+        <span class="desktop"
+          ><br />Culpa vitae consectetur, quo autem architecto ipsa aliquam
+          fugit numquam ipsum. Pariatur debitis non modi, accusantium eligendi
+          commodi iste vel sequi et! Lorem ipsum dolor sit amet consectetur
+          adipisicing elit.<br /><br />Repellat ratione beatae ipsam corporis
+          minima assumenda natus numquam sapiente ea excepturi, ab culpa
+          voluptate.Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Quod, unde? Modi quasi facilis sapiente reprehenderit?</span
+        >
+      </p>
+      <div class="contact-us" id="contact">
+        <p class="contact about">
+          Please contact us per Email for any further questions about the Great British Cattle Bazaar 2027!
+        </p>
+        <a href="#" class="contact-site about">GBCattleB2027@gbcb.com/beef</a>
+      </div>
+    </section>
+    <section class="logo-about">
+      <h2>Great British<br class="mobile"/>Cattle Bazaar 2027 Logo</h2>
+      <p>
+        The logo of the Great British Cattle Bazaar 2027 was amet consectetur adipisicing
+        elit. Lorem ipsum dolor sit met consectetur.<span class="desktop"><br>Lorem ipsum dolor sit emat, consectetur adipisicing elit. A ducimus, minus numquam omnis dignissimos nobis eius obcaecati veritatis vitae laudantium sint harum adipisci! Iusto sint iure eligendi, ullam corrupti ipsum.</span>
+      </p>
+      <div class="logo1">
+        <img src="resources/images/logo.png" alt="summit logo" />
+      </div>
+    </section>
+    <section class="past-about">
+      <h2>See the past Great British Cattle Bazaars</h2>
+      <p>
+        Take a look at the last two amet consectetur adipisicing elit. Lorem
+        ipsum dolor sit amet consectetur and amet.
+      </p>
+      <section class="past-events">
+
+        <div class="past-cc" id="cc2014">
+          2026<br /><span>Great British Cattle Bazaar <br />in Buenos Aires</span>
+        </div>
+        <div class="past-cc" id="cc2013">
+          2025<br /><span>Great British Cattle Bazaar <br />in Berlin</span>
+        </div>
+      </section>
+    </section>
+    <section class="partners mobile">
+      <h2>Partners</h2>
+      <div class="partner-logos">
+        <img src="resources/images/behance.png" alt="icon"/>
+        <img src="resources/images/dropbox.png" alt="icon2"/>
+        <img src="resources/images/soundcloud.png" alt="icon3"/>
+        <img src="resources/images/ebay.png" alt="icon4"/>
+        <img src="resources/images/wordpress.png" alt="icon5"/>
+        <img src="resources/images/linkedin.png" alt="icon6"/>
+        <img src="resources/images/instagram.png" alt="icon7"/>
+      </div>
+    </section>
+    <footer class="about">
```

Comment:
- [ ] The footer in the about page should look like this one, please update it:
![Screenshot from 2021-11-19 20-07-03](https://user-images.githubusercontent.com/62705947/142677907-af50d5f6-16c6-4fbf-8d81-2906954073f7.png)